priv/repo/migrations/00000000000001_create_user.exs

```diff
@@ -9,7 +9,6 @@ defmodule Org.Repo.Migrations.CreateUser do
       add :login, :string
       add :company, :string
       add :location, :string
-      add :created_at, :string
```

Comment:
I'll just revert this and add a comment in the model line explaining that it is the github user `created_at` date 👍